python/paddle/nn/functional/activation.py

```diff
@@ -774,7 +774,7 @@ def relu(x, name=None):
 def relu_(x, name=None):
     """
     Inplace version of ``relu`` API, the output Tensor will be inplaced with input ``x``.
-    Please refer to :ref:`api_nn_cn_relu`.
+    Please refer to :ref:`api_paddle_nn_ReLU`.
```

Comment:
同理，functional 下的 inplace api，refer 的应该也是functional下的原API
```suggestion
    Please refer to :ref:`api_paddle_nn_functional_relu`.
```